src/sentry/notifications/utils/__init__.py

```diff
@@ -412,10 +417,21 @@ def get_span_and_problem(
     return (spans, matched_problem)
 
 
-def get_transaction_data(event: Event) -> Any:
+def get_transaction_data(event: Event, project: Project) -> Any:
     """Get data about a transaction to populate alert emails."""
-    spans, matched_problem = get_span_and_problem(event)
-    return perf_to_email_html(spans, matched_problem, event)
+    if options.get(
+        "performance.issues.create_issues_through_platform", True
+    ) and project.get_option("sentry:performance_issue_create_issue_through_platform", True):
```

Comment:
Another option here is to check if there's an occurrence present and if so that implies the other options have been enabled